src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -24,15 +36,26 @@ public GuiSettings() {
         windowWidth = DEFAULT_WIDTH;
         windowHeight = DEFAULT_HEIGHT;
         windowCoordinates = null; // null represent no coordinates
```

Comment:
Can we possibly make a NULL_COORDINATE here for safer handling of vars?

Reply:
Sure, I have added that. Please take a look once I fixed the failing pipeline